server/runtime_status.go

```diff
@@ -27,12 +28,31 @@ func (s *Server) Status(ctx context.Context, req *types.StatusRequest) (*types.S
 		networkCondition.Message = fmt.Sprintf("Network plugin returns error: %v", err)
 	}
 
-	return &types.StatusResponse{
+	resp := &types.StatusResponse{
 		Status: &types.RuntimeStatus{
 			Conditions: []*types.RuntimeCondition{
 				runtimeCondition,
 				networkCondition,
 			},
 		},
-	}, nil
+	}
+	if req.Verbose {
+		info, err := createRuntimeInfo(s)
+		if err != nil {
+			return nil, fmt.Errorf("creating runtime info: %w", err)
+		}
+		resp.Info = info
+	}
+	return resp, nil
+}
+
+func createRuntimeInfo(s *Server) (map[string]string, error) {
+	config := map[string]interface{}{
+		"sandboxImage": s.config.ImageConfig.PauseImage,
+	}
+	bytes, err := json.Marshal(config)
+	if err != nil {
+		return nil, fmt.Errorf("marshal data: %v: %w", config, err)
```

Comment:
I don't think we need the whole config in the error message, but I'm not blocking on that.

Reply:
I was trying to do the same as with the other verbose "info" fields, for containers and images